src/_includes/components/apply-note.njk

```diff
@@ -0,0 +1,21 @@
+<div>
+  <div>Octogons shapes go here</div>
+  <div>
+    <p>When you’re ready to apply we need two things:</p>
+    <div>
```

Comment:
Since this is a list I think a `ol` would better suit it.

Reply:
Fixed!